src/main/controllers/SubscriptionConfirmedController.ts

```diff
@@ -0,0 +1,14 @@
+import { Response } from 'express';
+import {PipRequest} from '../models/request/PipRequest';
+import {cloneDeep} from 'lodash';
+
+export default class SubscriptionConfirmedController {
+
+  public async get(req: PipRequest, res: Response): Promise<void> {
+
+    res.render('subscription-confirmed', {
+      ...cloneDeep(req.i18n.getDataByLanguage(req.lng)['subscription-confirmed']),
```

Comment:
with no other options provided you can just call this inline, theres no need to deepen the params
`res.render('subscription-confirmed', req.i18n.getDataByLanguage(req.lng)['subscription-confirmed']);`

Reply:
this has been refactored

Reply:
this has been refactored

Reply:
this has been refactored